app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+
+  def new
+
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.authenticate(params[:session][:password])
+      log_in(user)
+      redirect_to user
```

Comment:
Rails handles it but it's better to pass a path instead of an active record object to `redirect_to` method.